bindings_ffi/src/mls.rs

```diff
@@ -352,13 +388,18 @@ impl FfiGroup {
             self.inner_client.as_ref(),
             self.group_id.clone(),
             self.created_at_ns,
+            self.added_by_address(),
         );
 
         group.sync().await?;
 
         Ok(())
     }
 
+    pub fn added_by_address(&self) -> Option<String> {
+        self.added_by.as_ref().map(|member| member.account_address.clone())
+    }
```

Comment:
Lets return the member here instead. I think it might be something like this

```suggestion
    pub fn added_by(&self) -> Option<FfiMember> {
        self.added_by.into()
    }
```

Reply:
Ethan and I talked about this and he's just going to remove this function in favor of the impl in the struct